README_EN.md

```diff
@@ -71,10 +71,51 @@ cd csghub-sdk
 pip install .
 ```
 
-## Use cases
+## Use cases of command line
+
+```shell
+# download model
+csghub download --repo-id=wanghh2000/myprivate1 --repo-type=model --token=3b77c98077b415ca381ded189b86d5df226e3776
```

Comment:
it's better to download a model like this

> csghub-cli download wanghh2000/myprivate1 

and download a dataset like this 

> csghub-cli download wanghh2000/myds1 --repo-type=dataset

Reply:
fixed.